atest/acceptance/keywords/cookies.robot

```diff
@@ -71,12 +71,12 @@ Get Cookies As Dict When There Are None
 
 Test Get Cookie Object Expiry
     ${cookie} =    Get Cookie      another
-    Should Be Equal As Integers    ${cookie.expiry.year}           2027
-    Should Be Equal As Integers    ${cookie.expiry.month}          09
-    Should Be Equal As Integers    ${cookie.expiry.day}            28
-    Should Be Equal As Integers    ${cookie.expiry.hour}           16
-    Should Be Equal As Integers    ${cookie.expiry.minute}         21
-    Should Be Equal As Integers    ${cookie.expiry.second}         35
+    Should Be Equal As Integers    ${cookie.expiry.year}           2023
```

Comment:
Because this is so near now, I would start creating it dynamically at some point. Same goes for all tests that set the expiry date, but can be done in separate PR.

Reply:
Will create a pr for this.